lib/board.rb

```diff
@@ -0,0 +1,39 @@
+class Board
+  def initialize
+    @board = { a1: ' ', a2: ' ', a3: ' ', b1: ' ', b2: ' ', b3: ' ', c1: ' ', c2: ' ', c3: ' ' }
+  end
+
+  def display_board
+    puts '  1 2 3'
+    puts "a|#{@board[:a1]}|#{@board[:a2]}|#{@board[:a3]}|"
+    puts "b|#{@board[:b1]}|#{@board[:b2]}|#{@board[:b3]}|"
+    puts "c|#{@board[:c1]}|#{@board[:c2]}|#{@board[:c3]}|"
+  end
+
+  def empty_cells
+    freecells = @board.select { |_k, v| v == ' ' }
+    print freecells.keys
+  end
+
```

Comment:
- [ ] Your `main.rb` file should be responsible for all user interactions. Kindly ensure that there are no `puts/print` and `get.chomp` outside `main.rb` file. If you must output some strings, you can assign it to variables and return them and then, call them appropriately in the `main.rb` file. Here is a [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from pathwright for clarification.